src/main/scala/uk/gov/nationalarchives/omega/api/repository/SparqlParams.scala

```diff
@@ -25,8 +25,8 @@ import org.apache.jena.rdf.model.{ Resource, ResourceFactory }
 import uk.gov.nationalarchives.omega.api.messages.request.ListAgentSummary
 import uk.gov.nationalarchives.omega.api.repository.model.AgentTypeMapper
 
-import java.text.SimpleDateFormat
-import java.util.GregorianCalendar
+import java.time.ZonedDateTime
+import java.util.{ Date, GregorianCalendar }
```

Comment:
Should we be mixing the old Java Date API with the new Java Time API. Is there a way to avoid the older API completely perhaps?

Reply:
Yes, turns out we don't need `java.util.Date` at all.